src/server/home/index.njk

```diff
@@ -1,42 +1,123 @@
+        from the Animal and Plant Health Agency (APHA). The pack was posted to
+        you when your herd was placed under TB restriction.
+      </p>
 
-        <p>You should not apply for a licence if:</p>
+      <p class="govuk-body">
+        For movements onto your farm or premises,
+        <a
+          href="https://www.gov.uk/government/publications/tb-restricted-cattle-application-for-movement-licence-in-england"
+          class="govuk-link"
+          >complete the TB204 form</a
+        >.
+      </p>
 
-        <ul class="govuk-body govuk-list--bullet">
-            <li>APHA has not received the results from cattle tuberculin skin tests or interferon-gamma blood tests</li>
-        </ul>
-      </div>
+      <h3 class="govuk-heading-m">Before you start</h3>
+
+      <p class="govuk-body">Licences cannot be assessed or issued if:</p>
+
+      <ul class="govuk-list govuk-list--bullet">
+        <li>reactors are still on your farm or premises</li>
+        <li>cattle are awaiting skin or blood test results</li>
+      </ul>
+
+      <p class="govuk-body">
+        Contact your vet if you are unsure whether the tests have been sent. If
+        APHA has not received the tests, there could be delays in processing
+        your licence.
+      </p>
+
+      <p class="govuk-body">
+        To apply for a licence, you need the following information:
+      </p>
+
+      <ul class="govuk-list govuk-list--bullet">
+        <li>name of the County Parish Holding (CPH) owner</li>
+        <li>
+          County Parish Holding (CPH) number and address where the animals are
+          moving from
+        </li>
+        <li>where the animals are moving to, for example the type of market</li>
+        <li>email address to send the licence to</li>
+      </ul>
+
+      <p class="govuk-body">
+        It can take up to 5 working days to process your application. It may
+        take longer if veterinary checks are required, or we need further
+        information.
+      </p>
+
+      <h3 class="govuk-heading-m">Get help</h3>
+
+      <p class="govuk-body">
+        If you need help using the service you can call 03000 200 301 and choose
+        the option for TB licencing. Lines are open from Monday to Friday
+        between 8.30am and 5pm.
+      </p>
+
+      <a
+        data-testid="start-now-btn"
+        class="gem-c-button govuk-button govuk-button--start"
+        role="button"
+        rel="external"
+        data-module="govuk-button"
+        draggable="false"
+        href="task-list"
+      >
+        Start now
+        <svg
+          class="govuk-button__start-icon govuk-!-display-none-print"
+          xmlns="http://www.w3.org/2000/svg"
+          width="17.5"
+          height="19"
+          viewBox="0 0 33 40"
+          focusable="false"
+          aria-hidden="true"
+        >
+          <path fill="currentColor" d="M0 0h13l20 20-20 20H0l20-20z"></path>
+        </svg>
+      </a>
```

Comment:
@joseluisgraa  - worth using the standard gov uk design system templates with this: 

```jinja2
{% from "govuk/components/button/macro.njk" import govukButton %}

{{ govukButton({
  text: "Start now",
  href: "#",
  isStartButton: true
}) }}
```

Reply:
https://design-system.service.gov.uk/components/button/ <- from here. 

The reason being the usual - we get all of the accessibility benefits out of the box, and don't have to worry about maintaining it as we upgrade the design system.